reana_workflow_controller/config.py

```diff
@@ -407,3 +407,12 @@ def _parse_interactive_sessions_environments(env_var):
 
 MAX_WORKFLOW_SHARING_MESSAGE_LENGTH = 5000
 """Maximum length of the user-provided message when sharing a workflow."""
+
+REANA_GC_ENABLED = int(os.getenv("REANA_GC_ENABLED", "0"))
+"""
+Garbage collection aggressiveness level. Valid values are integers from 0 to 9:
+(O: GC disabled, 9: most aggressive GC)
+"""
+
+REANA_MAX_WORKSPACE_FILES = int(os.getenv("REANA_MAX_WORKSPACE_FILES", "100000"))
```

Comment:
The name `REANA_MAX_WORKSPACE_FILES` could seem to imply that a workspace cannot have more than MAX workspace files, while it actually can. (We are only limiting the endpoint reply.)

What about inventing another name that would be more softer? Perhaps `WORKSPACE_FILES_DISPLAY_LIMIT` or `WORKSPACE_DISPLAY_FILE_LIMIT` or somesuch? We can muse IRL about a nice appropriate name...